build-tools/packages/build-tools/src/common/npmPackage.ts

```diff
@@ -207,9 +208,10 @@ export class Package {
 	}
 
 	public async savePackageJson() {
-		updatePackageJsonFile(this.directory, () => {
-			return;
-		});
+		return writeFileAsync(
```

Comment:
this is duplicating logic to write package.json files we have elsewhere (in updatePackageJsonFile) and failing to implement all the logic we had there (like getting the correct indent).

Instead factor out a function for writing package.json files, and call that.